$OpenBSD: patch-source_tag_c,v 1.1 2012/06/29 09:16:23 dcoppa Exp $

Fix segfault when sending empty tag strings
(upstream git commit be44e166c40e9708dbee563dbe6915a5391fe75c)

--- source/tag.c.orig	Fri Jun 29 11:07:52 2012
+++ source/tag.c	Fri Jun 29 11:08:28 2012
@@ -222,7 +222,7 @@ void sendtag(char key, char * tagstring, struct hash d
 	if(tagstring) {
 		unsigned length = strlen(tagstring);
 		/* remove trailing commas */
-		while(tagstring[length-1] == ',')
+		while(length > 0 && tagstring[length-1] == ',')
 			tagstring[--length] = 0;
 
 		splt = split(tagstring, ",\n", & nsplt);
